src/controllers/qasm_controller.hpp

```diff
@@ -298,6 +298,9 @@ class QasmController : public Base::Controller {
 
   // Controller-level parameter for CH method
   bool extended_stabilizer_measure_sampling_ = false;
+
+  // Disable SIMD optimization for testing
+  bool disable_simd_ = false;
```

Comment:
Is it really necessary to build this into the controller? It greatly complicates the code if it is only for testing. Why would anyone want to disable SIMD?